packages/peregrine/lib/talons/AuthModal/useAuthModal.js

```diff
@@ -35,14 +36,15 @@ const UNAUTHED_ONLY = ['CREATE_ACCOUNT', 'FORGOT_PASSWORD', 'SIGN_IN'];
 export const useAuthModal = props => {
     const {
         closeDrawer,
+        mutations = DEFAULT_MUTATIONS,
```

Comment:
I like namespacing the queries and mutations. IMO it could be either:

```js
const { mutations, queries } = props;
const { operations } = props; // where operations is { mutations, queries }
```

Both would allow users to customize specific named operations through the passed props by importing the default and spreading over a new object.